src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -745,7 +775,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		}));
 
 		this._register(dom.addDisposableListener(xterm.element, dom.EventType.DROP, (e: DragEvent) => {
-			this._onDropped.fire({ instance: this, dragEvent: e });
+			this._onDropped.fire(e);
 		}));
```

Comment:
Let's inline the handler here and remove the event, I don't think we will ever want to expose this